servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/RequestTagExtractor.java

```diff
@@ -32,15 +34,22 @@ private static String getRequestMethod(HttpRequestMetaData req) {
     }
 
     private static String getHttpUrl(HttpRequestMetaData req) {
-        return (req.scheme() == null ? "http" : req.scheme()) + "://" +
-            (req.effectiveHostAndPort() == null ? "localhost:8080" : req.effectiveHostAndPort())
-            + req.rawPath()
+        return req.path()
```

Comment:
does this equal the code you removed, or is this a behavioral change for the tag value? If so, is this acceptable?

Reply:
behavioral change.